README.md

```diff
@@ -104,7 +110,6 @@ In addition, integration modules are available for the following libraries, prov
 
 ### Examples of Distributed Optimization
 
-* [Optimizing on a Dask cluster](./dask/dask_simple.py)
```

Comment:
[question] Why do you remove this one?

Reply:
Oh... thank you for pointing it out. I revert it.